stubs/pywin32/_win32typing.pyi

```diff
@@ -0,0 +1,6661 @@
+# Not available at runtime. Contains type definitions that are otherwise not exposed
+from _typeshed import Incomplete
+from typing_extensions import final
+
+class PyDSCAPSType:
+    def __new__(cls): ...
```

Comment:
What's the point of these `__new__` methods in the stubs? Wouldn't the stub behave the same without them?

Reply:
Pretty sure there's not point and they're just leftovers from copying the original stubs